src/store/plugins/pendingTransactionHandler.js

```diff
@@ -0,0 +1,32 @@
+export default store => {
+  const waitTransactionMined = async ({ hash, type, amount, tipUrl }) => {
+    try {
+      const transaction = await store.state.sdk.poll(hash);
+      switch (type) {
+        case 'tip':
+          store.dispatch('router/push', { name: 'success-tip', params: { amount, tipUrl } });
+          break;
+        case 'spend':
+          store.dispatch('router/push', {
+            name: 'send',
+            params: { redirectstep: 3, successtx: transaction },
```

Comment:
I have misunderstood this part and instead `transaction` here was `true` that was causing the exception:
> Cannot read property 'amount' of undefined